extensions/wikia/WallNotifications/WallNotificationControllerBase.class.php

```diff
@@ -91,7 +91,7 @@ public function Notification() {
 
 		$msg = "";
 		wfRunHooks( 'NotificationGetNotificationMessage', [
-			&$this, &$msg, $firstEntity->isMain(), $data, $authors, $userCount, $wg->User->getName()
+			$this, &$msg, $firstEntity->isMain(), $data, $authors, $userCount, $wg->User->getName()
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this code to rely on RequestContext instead of context-dependent globals. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)